diff --git a/node_modules/expo-dynamic-image-crop/lib/hooks/usePerformCrop.js b/node_modules/expo-dynamic-image-crop/lib/hooks/usePerformCrop.js
index fe84bf2..c3874bc 100644
--- a/node_modules/expo-dynamic-image-crop/lib/hooks/usePerformCrop.js
+++ b/node_modules/expo-dynamic-image-crop/lib/hooks/usePerformCrop.js
@@ -20,8 +20,8 @@ export const usePerformCrop = () => {
             };
             setProcessing(true);
             const cropResult = await ImageManipulator.manipulateAsync(imageData.uri, [
-                { crop: croppingBounds },
-            ]);
+                { crop: croppingBounds }
+            ], {format : 'png'});
             const { uri, width, height } = cropResult;
             setImageData({ uri, width, height });
             setProcessing(false);
